addons/stock/models/stock_quant.py

```diff
@@ -358,13 +361,43 @@ def _compute_complete_name(self):
         return res
 
     def _compute_current_picking_info(self):
+        """ When a package is in displayed in picking, it gets the picking id trough the context, and this function
+        populates the different fields used when we move entire packages in pickings.
+        """
+        for package in self:
+            picking_id = self.env.context.get('picking_id')
+            if not picking_id:
+                package.current_picking_move_line_ids = False
+                package.current_picking_id = False
+                package.is_processed = False
+                package.current_source_location_id = False
+                package.current_destination_location_id = False
+                continue
+            package.current_picking_move_line_ids = package.move_line_ids.filtered(lambda move_line: move_line.picking_id.id == picking_id)
+            package.current_picking_id = True
+            package.current_source_location_id = package.current_picking_move_line_ids[:1].location_id
+            package.current_destination_location_id = package.current_picking_move_line_ids[:1].location_dest_id
+            if package.current_picking_move_line_ids.filtered(lambda ml: ml.qty_done < ml.product_uom_qty):
```

Comment:
Better to make `package.is_processed = bool(package.current_picking_move_line_ids.filtered(lambda ml: ml.qty_done < ml.product_uom_qty))` in one line